### DISCUSSION TITLE
Minimal docker image

### DISCUSSION THREAD

**Discussion by ZackarySantana** [apple/pkl#199] [Q&A]
Is there a minimal docker image for ci/cd tasks?

**Comment by holzensp** [comment]
There isn't. It's not clear how much we will be able to support from the core team. Maybe this might be easier to get moving with https://github.com/pkl-community.

**Comment by jesscarlos** [comment]
@ZackarySantana I had the same need, I created a simple docker image to run the CLI. 

GitHub: https://github.com/ignittion/pkl-cli
Docker Hub: https://hub.docker.com/r/jesscarlos/pkl-cli
